packages/squiggle-lang/__tests__/Distributions/AlgebraicCombination_test.res

```diff
@@ -0,0 +1,313 @@
+open Jest
+open Expect
+open TestHelpers
+
+let {
+  normalDist5,  // mean=5, stdev=2
+  normalDist10,  // mean=10, stdev=2
+  normalDist20,  // mean=20, stdev=2
+  normalDist,  // mean=5; stdev=2
+  uniformDist,  // low=9; high=10
+  betaDist,  // alpha=2; beta=5
+  lognormalDist,  // mu=0; sigma=1
+  cauchyDist,  // local=1; scale=1
```

Comment:
if you'll need comments, it could be easier to just call these things like ``normalDist_m5std2``